Makefile

```diff
@@ -247,6 +243,9 @@ endif
 	@echo
 	@echo "[Success] Renga UI should be under $(RENGA_UI_URL) and GitLab under $(GITLAB_URL)"
 	@echo
+	@echo "[Info] If running locally, make sure $(PLATFORM_DOMAIN) keycloak.$(PLATFORM_DOMAIN) and gitlab.$(PLATFORM_DOMAIN)"
```

Comment:
Can we generate a command to run only if the names are not defined in `/etc/hosts` already (e.g. `echo "127.0.0.1 $(PLAFORM_DOMAIN) ... >> /etc/hosts`)?

Reply:
you mean just to show it in the info?

Reply:
Yes. Also we should only show the info if the configuration is wrong (e.g. you can't ping/traceroute the name).